### PR TITLE
[change] set default values for the config properties

### PR DESCRIPTION
**Pull request by dacrome**

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -81,31 +85,25 @@ Here some examples:
...
 
-#### org.osiam.mail.server.* (with examples, mostly self explained
-* *.smtp.port=25
-* *.host.name=http://www.example.org
-* *.username=username
-* *.password=password
-* *.smtp.starttls.enable=true
-* *.smtp.auth=true
-* *.transport.protocol=smtp
+#### org.osiam.mail.server.* (with defaults/examples, mostly self explained
```
Comment (tkrille) [inline docs/configuration.md]: you also have to change the TOC if you change the heading

File: docs/configuration.md [hunk 2]
```diff
@@ -81,31 +85,25 @@ Here some examples:
...
+* *.smtp.port=25 (default)
+* *.host.name=localhost (default)
+* *.username=username (example, do not have to be set)
+* *.password=password (example, do not have to be set)
+* *.smtp.starttls.enable=true (example, do not have to be set)
+* *.smtp.auth=true (example, do not have to be set)
+* *.transport.protocol=smtp (example, do not have to be set)
```
Comment (tkrille) [inline docs/configuration.md]: these are not representing the default values.

File: docs/configuration.md (hunk 1 above)
Comment (dacrome) [inline docs/configuration.md]: then I should move the description in the next line

Comment (tkrille) [inline docs/configuration.md]: or change the TOC :) I'm good with both.

File: src/main/deploy/addon-self-administration.properties [hunk 3]
```diff
@@ -26,58 +24,61 @@ org.osiam.mail.emailchange.linkprefix=http://localhost/
...
-org.osiam.mail.server.username=username
-org.osiam.mail.server.password=password
-org.osiam.mail.server.smtp.starttls.enable=true
-org.osiam.mail.server.smtp.auth=true
+org.osiam.mail.server.host.name=localhost
 org.osiam.mail.server.transport.protocol=smtp
+#org.osiam.mail.server.username=username
+#org.osiam.mail.server.password=password
+#org.osiam.mail.server.smtp.starttls.enable=true
+#org.osiam.mail.server.smtp.auth=true
```
Comment (tkrille) [inline src/main/deploy/addon-self-administration.properties]: these are not representing the default values.

File: src/main/deploy/addon-self-administration.properties [hunk 4]
```diff
@@ -1,10 +1,8 @@
-# OSIAM auth server configuration
-org.osiam.auth-server.home=http://localhost:8080/osiam-auth-server
-
-# OSIAM resource server configuration
-org.osiam.resource-server.home=http://localhost:8080/osiam-resource-server
+# Please sync the client data with the ../sql/example_data.sql
```
Comment (tkrille) [inline src/main/deploy/addon-self-administration.properties]: this is now "../sql/client.sql". I changed this [here](https://github.com/osiam/addon-self-administration/pull/112/files#diff-6fd6cb25e5f1cae18884cf8ef42981f8L40), but you moved the whole block upwards.

File: src/main/java/org/osiam/addons/self_administration/Config.java [hunk 5]
```diff
@@ -23,6 +23,30 @@
 
     public static final String CONFIRMATION_TOKEN_FIELD = "emailConfirmToken";
 
+    @Value("${org.osiam.mail.from}")
```
Comment (tkrille) [inline src/main/java/org/osiam/addons/self_administration/Config.java]: no default value set

**Pull request merged**